zh/backup-to-aws-s3.md

```diff
@@ -0,0 +1,516 @@
+---
+title: 使用 BR 工具备份 AWS 上的 TiDB 集群
+category: how-to
+---
+
+# 使用 BR 工具备份 AWS 上的 TiDB 集群
+
+本文详细描述了如何将运行在 AWS Kubernetes 环境中的 TiDB 集群数据备份到 AWS 的存储上。本文档中的“备份”，均是指全量备份（Ad-hoc 全量备份和定时全量备份）。底层通过使用 [`br`](https://pingcap.com/docs-cn/v3.1/reference/tools/br/br) 获取集群的逻辑备份，然后在将备份数据上传到 AWS 的存储上。
+
+本文使用的备份方式基于 TiDB Operator 新版（v1.1 及以上）的 CustomResourceDefinition (CRD) 实现。
+
+## AWS 账号的三种权限授予方式
+
+在 AWS 云环境中， 不同的类型的 kubernetes 集群提供了不同的权限授予方式。本文测试了以下三种权限授予方式。
+
+1. 通过传入 AWS 账号的 AccessKey 和 SecretKey 进行授权:
+
+    AWS 的客户端支持读取进程环境变量中的 `AWS_ACCESS_KEY_ID` 以及 `AWS_SECRET_ACCESS_KEY` 来获取与之相关联的用户或者角色的权限。
+
+2. 通过将 [`IAM`](https://aws.amazon.com/cn/iam/) 绑定 `Pod` 进行授权:
+
+    通过将用户的 `IAM` 角色与所运行的 `Pod` 资源进行绑定，使 `Pod` 中运行的进程获得角色所拥有的权限, 这种授权方式是由 [`kube2iam`](https://github.com/jtblin/kube2iam) 提供。
+
+> **注意：**
+>
+> - 使用该授权模式时，可以参考[`kube2iam 文档`](https://github.com/jtblin/kube2iam#usage) 在 kubernetes 集群中创建 kube2iam 环境, 并且部署 TiDB Operator 以及 TiDB 集群。
+> - 该模式不适用与 [`hostNetwork`](https://kubernetes.io/docs/concepts/policy/pod-security-policy) 网络模式，请确保参数 `spec.tikv.hostNetwork` 参数为 `false`。
+
+3. 通过将 [`IAM`](https://aws.amazon.com/cn/iam/) 绑定 `ServiceAccount` 进行授权:
+
+    通过将用户的 `IAM` 角色与 kubeneters 中的 [`serviceAccount`](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) 资源进行绑定, 从而使得使用该 `ServiceAccount` 账号的 `Pod` 都拥有该角色所拥有的权限，这种授权方式由 [`EKS`](https://aws.amazon.com/cn/eks/) 服务提供。
+
+> - 使用该授权模式时，可以参考[`AWS 官方文档`](https://docs.aws.amazon.com/zh_cn/eks/latest/userguide/create-cluster.html) 创建 eks 集群, 并且部署 TiDB Operator 以及 TiDB 集群。
```

Comment:
```suggestion
> - 使用该授权模式时，可以参考 [AWS 官方文档](https://docs.aws.amazon.com/zh_cn/eks/latest/userguide/create-cluster.html)创建 eks 集群，并且部署 TiDB Operator 以及 TiDB 集群。
```

Reply:
Can we remove this sentence, it does not have any value for this doc.